NRzure/quickstarts/example-quickstart/config.yml

```diff
@@ -0,0 +1,51 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: AzureAD
+
+# Displayed in the UI (required)
+title: Azure Active Directory
+
+# Long-form description of the quickstart (required)
+description: |
+  This quickstart allows for basic data collection about information related to the Azure Active Directory services.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Basic data collection about Azure Active Directory.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Liz Huller
+  - Will Foster
+  - Jonathan Chu
+  - Daphne Hong
+  - Hala Rawas
+  - Momo Siu
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - azure
+  - active directory
+  - azure ad
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - example-install
+alerts:
+  - condition-a
```

Comment:
To link to the current alerts you have setup, this would need to be `example-alert-policy` and you could remove line 39. However, let's rename that alert directory to something more descriptive!